PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -359,13 +423,9 @@
         }
 
         private func isSwiftUIImageSensitive(_ view: UIView) -> Bool {
-            // the raw type _UIGraphicsView is always something like Color.white or similar
-            // never contains PII and should not be masked
-            // Button will fall in this case case but Button has subviews
-            let type = type(of: view)
-
-            let rawGraphicsView = String(describing: type) == "_UIGraphicsView"
-            return (config.sessionReplayConfig.maskAllImages || view.isNoCapture()) && !rawGraphicsView
+            // No way of checking if this is an asset image or not
+            // No way of checking if there's actual content in the image or not
```

Comment:
the previous check was working before, I recall testing this, so why isn't it anymore?